cloudify_nsx/network/dlr.py

```diff
@@ -32,15 +32,9 @@ def create(**kwargs):
         "dlr_size": {
             "required": True
         },
-        "datacentermoid": {
-            "required": True
-        },
-        "datastoremoid": {
-            "required": True
-        },
-        "resourcepoolid": {
-            "required": True
-        },
+        "datacentermoid": {},
```

Comment:
I actually left these in so that either approach can be used (e.g. it can be configured without using any vsphere objects). Is that not intended?